<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>666</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QPushButton" name="btnCrear">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>20</y>
      <width>71</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Crear</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btn_Modificar">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>50</y>
      <width>71</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Modificar</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btnObtener">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>80</y>
      <width>71</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Obtener</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btnDiferencia">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>110</y>
      <width>71</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Diferencia</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btnHistorial">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>140</y>
      <width>71</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Historial</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="fieldCrear">
    <property name="geometry">
     <rect>
      <x>130</x>
      <y>20</y>
      <width>113</width>
      <height>20</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="fieldObtener">
    <property name="geometry">
     <rect>
      <x>130</x>
      <y>20</y>
      <width>113</width>
      <height>20</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="fieldDiferencia">
    <property name="geometry">
     <rect>
      <x>130</x>
      <y>20</y>
      <width>113</width>
      <height>20</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="okCrear">
    <property name="geometry">
     <rect>
      <x>250</x>
      <y>20</y>
      <width>51</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>OK</string>
    </property>
   </widget>
   <widget class="QPushButton" name="okModificar">
    <property name="geometry">
     <rect>
      <x>500</x>
      <y>20</y>
      <width>51</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>OK</string>
    </property>
   </widget>
   <widget class="QPushButton" name="okObtener">
    <property name="geometry">
     <rect>
      <x>250</x>
      <y>20</y>
      <width>51</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>OK</string>
    </property>
   </widget>
   <widget class="QPushButton" name="okDiferencia">
    <property name="geometry">
     <rect>
      <x>380</x>
      <y>20</y>
      <width>51</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>OK</string>
    </property>
   </widget>
   <widget class="QLabel" name="msjCrear">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>50</y>
      <width>241</width>
      <height>61</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>91</width>
      <height>31</height>
     </size>
    </property>
    <property name="font">
     <font>
      <pointsize>22</pointsize>
     </font>
    </property>
    <property name="text">
     <string>¡Creación Exitosa!</string>
    </property>
   </widget>
   <widget class="QLabel" name="mErrorD">
    <property name="geometry">
     <rect>
      <x>130</x>
      <y>190</y>
      <width>461</width>
      <height>91</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>10</pointsize>
     </font>
    </property>
    <property name="text">
     <string>¡Debe crear un programa primero!</string>
    </property>
   </widget>
   <widget class="QLabel" name="mErrorT">
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>220</y>
      <width>51</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>10</pointsize>
     </font>
    </property>
    <property name="text">
     <string>ERROR:</string>
    </property>
   </widget>
   <widget class="QTextEdit" name="fieldModificar">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>20</y>
      <width>371</width>
      <height>171</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="selectVersion">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>20</y>
      <width>113</width>
      <height>20</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="okModificar2">
    <property name="geometry">
     <rect>
      <x>240</x>
      <y>20</y>
      <width>51</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>OK</string>
    </property>
   </widget>
   <widget class="QLabel" name="msjMod">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>40</y>
      <width>191</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Ingrese la version que desea modificar</string>
    </property>
   </widget>
   <widget class="QLabel" name="tInfoMod">
    <property name="geometry">
     <rect>
      <x>170</x>
      <y>10</y>
      <width>371</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>18</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Información de la modificación</string>
    </property>
   </widget>
   <widget class="QTextEdit" name="infoMod">
    <property name="enabled">
     <bool>false</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>160</x>
      <y>50</y>
      <width>331</width>
      <height>171</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Times New Roman</family>
      <pointsize>14</pointsize>
     </font>
    </property>
   </widget>
   <widget class="QTextEdit" name="infoVer">
    <property name="enabled">
     <bool>false</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>160</x>
      <y>50</y>
      <width>331</width>
      <height>171</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="tVersion">
    <property name="geometry">
     <rect>
      <x>210</x>
      <y>0</y>
      <width>231</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>16</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Código de la versión</string>
    </property>
   </widget>
   <widget class="QLabel" name="errorOb">
    <property name="geometry">
     <rect>
      <x>140</x>
      <y>40</y>
      <width>431</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>9</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Error msg</string>
    </property>
   </widget>
   <widget class="QLabel" name="errorMod">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>40</y>
      <width>331</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>11</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Error msj</string>
    </property>
   </widget>
   <widget class="QLabel" name="errorCrear">
    <property name="geometry">
     <rect>
      <x>130</x>
      <y>40</y>
      <width>431</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>10</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Error msj</string>
    </property>
   </widget>
   <widget class="QTextEdit" name="fieldHist">
    <property name="enabled">
     <bool>false</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>160</x>
      <y>50</y>
      <width>331</width>
      <height>171</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>11</pointsize>
     </font>
    </property>
   </widget>
   <widget class="QLabel" name="titleHist">
    <property name="geometry">
     <rect>
      <x>180</x>
      <y>10</y>
      <width>311</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>20</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Historial de versiones</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="fieldDif2">
    <property name="geometry">
     <rect>
      <x>250</x>
      <y>20</y>
      <width>113</width>
      <height>20</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="infoDif1">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>40</y>
      <width>141</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Ingrese la version 1</string>
    </property>
   </widget>
   <widget class="QLabel" name="infoDif2">
    <property name="geometry">
     <rect>
      <x>260</x>
      <y>40</y>
      <width>151</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Ingrese la version 2</string>
    </property>
   </widget>
   <widget class="QTextEdit" name="showDifs">
    <property name="enabled">
     <bool>true</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>160</x>
      <y>50</y>
      <width>331</width>
      <height>171</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>11</pointsize>
     </font>
    </property>
   </widget>
   <widget class="QLabel" name="titleDif">
    <property name="geometry">
     <rect>
      <x>240</x>
      <y>20</y>
      <width>241</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>16</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Diferencias</string>
    </property>
   </widget>
   <widget class="QLabel" name="errorDif">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>50</y>
      <width>511</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>11</pointsize>
      <italic>true</italic>
     </font>
    </property>
    <property name="cursor">
     <cursorShape>UpArrowCursor</cursorShape>
    </property>
    <property name="text">
     <string>Error msj</string>
    </property>
   </widget>
   <zorder>btnCrear</zorder>
   <zorder>btn_Modificar</zorder>
   <zorder>btnObtener</zorder>
   <zorder>btnDiferencia</zorder>
   <zorder>btnHistorial</zorder>
   <zorder>fieldCrear</zorder>
   <zorder>fieldObtener</zorder>
   <zorder>fieldDiferencia</zorder>
   <zorder>okCrear</zorder>
   <zorder>okModificar</zorder>
   <zorder>okObtener</zorder>
   <zorder>msjCrear</zorder>
   <zorder>mErrorD</zorder>
   <zorder>mErrorT</zorder>
   <zorder>fieldModificar</zorder>
   <zorder>selectVersion</zorder>
   <zorder>okModificar2</zorder>
   <zorder>msjMod</zorder>
   <zorder>tInfoMod</zorder>
   <zorder>infoMod</zorder>
   <zorder>infoVer</zorder>
   <zorder>tVersion</zorder>
   <zorder>errorOb</zorder>
   <zorder>errorMod</zorder>
   <zorder>errorCrear</zorder>
   <zorder>fieldHist</zorder>
   <zorder>titleHist</zorder>
   <zorder>okDiferencia</zorder>
   <zorder>fieldDif2</zorder>
   <zorder>infoDif1</zorder>
   <zorder>infoDif2</zorder>
   <zorder>showDifs</zorder>
   <zorder>titleDif</zorder>
   <zorder>errorDif</zorder>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>666</width>
     <height>25</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
